src/llhttp/http.ts

```diff
@@ -594,7 +595,15 @@ export class HTTP {
     n('chunk_size')
       .select(HEX_MAP, this.mulAdd('content_length', {
```

Comment:
Let's move `this.mulAdd('content_length', ...)` to a shared declaration like this:

```js
const addContentLength = this.mulAdd('content_length', { ... });

n('chunk_size').select(HEX_MAP, addContentLength)...
n('chunk_size2').select(HEX_MAP, addContentLength)...
```